src/components/RepoSubmission.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useState } from 'react'
+
+
+const RepoSubmission: FC = ()  => {
```

Comment:
Why use FC here? Can you provide contexT?

Reply:
Here is a discussion on it that you can checkout
https://stackoverflow.com/a/59991281